### PR TITLE
feat: Add query params to find, findAll and findMany methods

### PR DESCRIPTION
**Pull request by nicobritos**
Add feature in #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,14 @@ var address = Address(await adapter.find('addresses', '1'));
 
 Will send the request: `GET /addresses/1`.
 
+#### Retrieving a relationship
+
+```dart
+var address = Address(await adapter.find('addresses', '1', queryParams: {'include', 'state'}));
+```
+
+Will send the request: `GET /addresses/1?include=state`.
```
Comment (Bringoff) [inline README.md:88]: @algodave not sure how our internal api will work with that, but I guess, it should be there

**Comment by algodave** [comment]
@nicobritos Thank you for this PR! I will take a closer look by the end of the current week

File: README.md (hunk 1 above)
Comment (algodave) [reply README.md:88]: @Bringoff The `include` query param is part of the `JSON:API` specifications, so it would be great to have our library support it 🙂

**Comment by algodave** [comment]
@nicobritos Awesome contribution, thank you! Sorry for taking a closer look so late

**Pull request merged**